epiprocess.qmd

```diff
@@ -35,20 +35,18 @@ Functions in the package that operate directly on given variables do not begin
 
 ## `epi_archive`: full version history of a data set
 
-The second main data structure in the package is called
-[`epi_archive`]. This is a special class (R6 format) 
-wrapped around a data table that stores the archive (version history) of some
-signal variables of interest.
+The second main data structure in the package is called [`epi_archive`]. This is
+an S3 class containing a data table that stores the archive (version history) of
+some signal variables of interest.
 
 By convention, functions in the `epiprocess` package that operate on
```

Comment:
```suggestion
By convention, functions in the `{epiprocess}` package that operate on
```

Probably a whac-a-mole exercise in futility, but this is the convention we _should_ use where possible.